### DISCUSSION TITLE
Database upgrade stuck ?

### DISCUSSION THREAD

**Discussion by manriif** [supabase/supabase#20300] [Questions]
Hi community,

First of all, i'd like to mention that I'm new to Supabase.

So, I have a little database of 50 tables that are currently empty or have at most one row. 
I've started an upgrade of my database 12 hours ago from the Dashboard and i'm here to ask if it is normal that the process is still at the "Executing pre-upgrade checks" step after so many hours ?

<img width="1247" alt="Screenshot 2024-01-10 at 10 43 20" src="https://github.com/supabase/supabase/assets/58949009/28839b54-f687-46a1-a3df-5b7c62b802c3">

The only particularity of my database is that i have other schemas than the public schema.

Thank you in advance for your answers.

**Comment by manriif** [comment]
From the support team, the upgrade was effectively blocked at the "Executing pre-upgrade checks" step. They rolled back the upgrade and are currently investigating the root cause.

**Comment by Sulomus** [comment]
Did they find root cause of this?

**Reply by manriif** [reply]
I think, not sure, below the exact sentence they’ve send me by email one month ago and one month after they’ve fixed my instance of the issue:

« We believe we have solved the underlying cause for this issue. »

Hope you are not facing the issue…
